apps/namadillo/src/hooks/useTransactionNotifications.tsx

```diff
@@ -357,11 +357,9 @@ export const useTransactionNotifications = (): void => {
       "UnshieldingTransfer.Error",
     ],
     (e) => {
-      const tx = e.detail.tx;
-      const data: TxWithAmount[] = e.detail.data[0].data;
-      const { id } = parseTxsData(tx, data);
+      const id = createNotificationId(e.detail.tx);
       clearPendingNotifications(id);
-      const storedTx = searchAllStoredTxByHash(tx.hash);
+      const storedTx = searchAllStoredTxByHash(e.detail.tx[0].hash);
```

Comment:
Please check if what you want is not the tx[0].innerTxHashes[0]

Reply:
Validated. The `tx[0].hash` is correct. We don't have the `innerTxHashes` on the event object, and we are creating the stored tx with the `txs[0].hash` from response, not the `txs[0].innerTxHashes[0]`

![Screenshot 2025-01-14 at 11 50 50](https://github.com/user-attachments/assets/960b3e33-893d-4d97-b3d3-06c8dba731c5)

![Screenshot 2025-01-14 at 11 51 04](https://github.com/user-attachments/assets/c1dbc643-678b-447f-800c-142c69aae1b3)